trio/_threads.py

```diff
@@ -59,7 +59,9 @@ class ThreadPlaceholder:
 
 
 @enable_ki_protection
-async def to_thread_run_sync(sync_fn, *args, cancellable=False, limiter=None):
+async def to_thread_run_sync(
+    sync_fn, *args, name: str = "", cancellable=False, limiter=None
```

Comment:
I prefer `thread_name=None` here, to make it clear what the name is for, and that we'll construct a default if not given a name.